safe_zone_width: 1.5
safe_zone_height: 1.5
controlled_zone_thickness: 1.0